plugins/GSdx/Renderers/HW/GSRendererHW.cpp

```diff
@@ -37,11 +37,13 @@ GSRendererHW::GSRendererHW(GSTextureCache* tc)
 	, m_userhacks_tcoffset_y(0)
 	, m_channel_shuffle(false)
 	, m_lod(GSVector2i(0,0))
+	, m_dithering(1)
```

Comment:
Isn't this line unnecessary if it gets reassigned a few lines later?

Reply:
I dunno, would need to verify it. I think I use copied `m_upscale_multiplier`.

Reply:
Confirmed unnecessary for `m_dithering` and `m_upscale_multiplier`. Should I remove both while at it?

Reply:
I'd say remove both if unnecessary.

Reply:
It's up to you.